### PR TITLE
Inorder Traversal Golang code for issue 415

### PR DESCRIPTION
**Pull request by Kathan-Vakharia**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added code for inorder traversal in Golang for issue #415 

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Comment by Kathan-Vakharia** [comment]
@Abhijit2505 , @Kajol-Kumari  , @plazzy99  kindly review my PR according to your convenience :)

**Comment by HarshCasper** [comment]
Looks good to me!

**Review by HarshCasper — APPROVED** [review]

**Comment by Kathan-Vakharia** [comment]
@plazzy99 @Kajol-Kumari  @siddharth25pandey ,Sorry to ping again but this PR has been approved 2 days ago . Can you merge it because I want to start another issue 😀

**Comment by siddharth25pandey** [comment]
> @plazzy99 @Kajol-Kumari @siddharth25pandey ,Sorry to ping again but this PR has been approved 2 days ago . Can you merge it because I want to start another issue 😀

@Kathan-Vakharia PR only be merged, if atleast 3 mentors will give approval

File: Go/ds/inorder_traversal_binarytree.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+func main() {
+	//Creating a Tree with following structure
+	/*
+			5
+		   / \
+		  6   7
+		 /   / \
+		3   10  9
+	*/
+	root := NewNode(5)
```
Comment (Kajol-Kumari) [inline Go/ds/inorder_traversal_binarytree.go]: Can you please take the node value input from user @Kathan-Vakharia ?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Please take the input dynamically,Rest LGTM :)

File: Go/ds/inorder_traversal_binarytree.go (hunk 1 above)
Comment (Kathan-Vakharia) [reply Go/ds/inorder_traversal_binarytree.go]: @Kajol-Kumari I have implemented your changes, Can you merge it now  :)

**Pull request merged**